clients/aws-sdk-transcribe-streaming/scripts/setup_resources.py

```diff
@@ -0,0 +1,93 @@
+# /// script
```

Comment:
This is great, thanks for putting it together! One minor thing is we may want to put this under the `tests/` directory. It may also be worth adding a quick disclaimer that this is for local testing but shouldn't be relied on in production setups.